README.md

```diff
@@ -152,6 +152,17 @@ cargo miri test
 When using the above instructions, you may encounter a number of confusing compiler
 errors.
 
+### "note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace"
+
+You may see this when trying to get Miri to display a backtrace. By default, Miri
+doesn't expose any environment to the program, so running
+`RUST_BACKTRACE=1 cargo miri test` will not do what you expect.
+
+To get a backtrace, you need to disable isolation
+[using `-Zmiri-disable-isolation`](#miri-flags):
+
+`RUST_BACKTRACE=1 MIRIFLAGS="-Zmiri-disable-isolation" cargo miri test`
```

Comment:
Please use triple-`` ` `` blocks when the whole paragraph should be code-formatted.